ui/k-info-panel/link_info.kytos

```diff
@@ -25,6 +25,16 @@
           </table>
         </div>
       </k-accordion-item>
+      <k-accordion-item title="Metadata actions">
+        <k-textarea title="Insert metadata" icon="arrow-right" placeholder='Eg. {"bandwidth": 100, "port_name": "some_name"}' :value.sync="to_add"></k-textarea>
```

Comment:
@Alopalao could you update the example in the placeholder to be `link_name` instead of `port_name` since it's related to a link?

Reply:
Placeholder updated on commit [308e6055141b855e9c60f269c813fea5e3535368](https://github.com/kytos-ng/topology/pull/79/commits/308e6055141b855e9c60f269c813fea5e3535368)